lambdas/es/indexer/test/test_index.py

```diff
@@ -171,6 +270,190 @@ def _get_contents(self, name, ext):
             etag='etag', version_id=None, s3_client=self.s3_client, size=123,
         )
 
+    def test_create_event_failure(self):
+        """
+        Check that the indexer doesn't blow up on create event failures.
+        """
+        # TODO, why does pytest.raises(RetryError not work?)
+        with pytest.raises(Exception, match="Failed to load"):
+            self._test_index_events(
+                ["ObjectCreated:Put"],
+                errors=True,
+                status=400
+            )
+
+    def test_create_index_events(self):
+        """test indexing a single file from a variety of create events"""
+        # test all known created events
+        # https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html
+        self._test_index_events(["ObjectCreated:Copy"])
+        self._test_index_events(["ObjectCreated:Put"], mock_elastic=False)
+        # Elastic only needs to be mocked once per test
+        self._test_index_events(["ObjectCreated:Post"], mock_elastic=False)
+        self._test_index_events(["ObjectCreated:CompleteMultipartUpload"])
+        self._test_index_events(["ObjectCreated:Put"], mock_elastic=False, bucket_versioning=False)
```

Comment:
`RequestsMock.stop()` asserts that mock responses were used at least once. So I think that instead of setting `mock_elastic=False` you should move all of these items in separate `test_xxx()` methods as it makes testing more rigorous.

Reply:
Yeah, lemme do that

Reply:
Done.